### PR TITLE
fix: switch to @rhoas/app-services-ui-shared

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
Checking attribute gen job

**Comment by wtrocki** [comment]
Package lock needs to be regenerated again

**Comment by wtrocki** [comment]
Would do that

**Comment by pmuir** [comment]
My bad - I optimistically moved to node 16

**Comment by wtrocki** [comment]
Still fails. My node is `v14.15.1`

**Review by wtrocki — APPROVED** [review]

File: .build/src/app/QuickStartContainer.tsx [hunk 1]
```diff
@@ -1,72 +1,68 @@
...
-}) => {
-  const [activeQuickStartID, setActiveQuickStartID] = useLocalStorage(
-    "quickstartId",
-    ""
-  );
-  const [allQuickStartStates, setAllQuickStartStates] = useLocalStorage(
-    "quickstarts",
-    {}
-  );
+                                                                              children,
```
[the remark this answers is not in the document]
Comment (pmuir) [reply .build/src/app/QuickStartContainer.tsx:21]: It's the default formatter in my IDE - let me fix it

[the remark this answers is not in the document]
Comment (pmuir) [reply .build/src/app/QuickStartContainer.tsx]: Fixed, also removed some unused code.

**Comment by wtrocki** [comment]
Run locally. Good to merge

**Pull request merged**